provisioners/configure-backend.sh

```diff
@@ -0,0 +1,4 @@
+./configure.sh
+echo "Configure Notification Service"
+envsubst < $TEMPLATES_PATH/etc/systemd/system/notification.service > /etc/systemd/system/notification.service
+systemctl enable notifications.service
```

Comment:
Please make sure there's a trailing newline at the end of the file! This should show up on `git diff` and such, as it does here on GitHub with a red dash in a circle.

This kind of thing should be caught in a linter, but we haven't configured linting for this repo yet.